mpc-recovery/src/lib.rs

```diff
@@ -91,9 +93,14 @@ pub enum Cli {
         /// Account creator ID
         #[arg(long, env("MPC_RECOVERY_ACCOUNT_CREATOR_ID"))]
         account_creator_id: AccountId,
-        /// TEMPORARY - Account creator ed25519 secret key
-        #[arg(long, env("MPC_RECOVERY_ACCOUNT_CREATOR_SK"))]
-        account_creator_sk: Option<String>,
+        /// TEMPORARY - Account creator ed25519 secret keys
```

Comment:
I don't think this is temporary anymore :grimacing:

Reply:
why was this a temporary solution in the first place?

Reply:
I think the idea was that we would always fetch the secret key from Secret Manager, but it's pretty evident that it is useful to also have as a CLI parameter

Reply:
I see. We may have too many entrypoints for data args:
- ENV variable
- CLI variable
- loading secrets from secrets manager

We could get rid of the loading secrets one as that has the least usage and terraform already loads them in via ENV variable already